webapp/components/Group/GroupForm.vue

```diff
@@ -7,133 +7,184 @@
-          :options="cities"
-          icon="map-marker"
-          :placeholder="$t('settings.data.labelCity') + ' …'"
-          :loading="loadingGeo"
-          @input.native="handleCityInput"
-        />
-        <!-- TODO: implement clear button -->
-        <!-- <base-button icon="close" circle ghost size="small" :disabled="formData.locationName.length === 0" @click="clear" /> -->
-      </ds-space>
-      <ds-space margin-top="large">
-        <categories-select
-          v-if="categoriesActive"
-          model="categoryIds"
-          :existingCategoryIds="formData.categoryIds"
-        />
-      </ds-space>
-      <ds-space margin-top="large">
-        <nuxt-link to="/my-groups">
-          <ds-button>{{ $t('actions.cancel') }}</ds-button>
-        </nuxt-link>
-        <ds-button
-          type="submit"
-          icon="save"
-          :disabled="update ? submitDisableEdit : submitDisable"
-          primary
-          @click.prevent="submit()"
-        >
-          {{ update ? $t('group.update') : $t('group.save') }}
-        </ds-button>
-      </ds-space>
+      <template #default="{ errors }">
+        <base-card>
+          <!-- Group Name -->
+          <ds-input
+            :label="$t('group.name')"
+            model="name"
+            name="name"
+            autofocus
+            :placeholder="`${$t('group.name')} …`"
+          />
+          <ds-text align="right">
+            <ds-chip size="base" :color="errors && errors.name && 'danger'">
+              {{ `${formData.name.length} / ${formSchema.name.max}` }}
+              <base-icon v-if="errors && errors.name" name="warning" />
+            </ds-chip>
+          </ds-text>
+
+          <!-- Group Slug -->
+          <ds-space margin-top="large">
+            <ds-input
+              v-if="update"
+              :label="$t('group.labelSlug')"
+              model="slug"
+              icon="at"
+              :placeholder="`${$t('group.labelSlug')} …`"
+            ></ds-input>
+          </ds-space>
+
+          <!-- groupType -->
+          <ds-space margin-top="large">
+            <ds-text class="select-label">
+              {{ $t('group.type') }}
+            </ds-text>
+            {{ formData.groupType }}
+            <select
+              class="select ds-input appearance--auto"
+              :value="formData.groupType"
+              :disabled="update"
+              model="groupType"
+              name="groupType"
+              @change="changeGroupType($event)"
+            >
+              <option v-for="groupType in groupTypeOptions" :key="groupType" :value="groupType">
+                {{ $t(`group.types.${groupType}`) }}
+              </option>
+            </select>
+            <ds-text align="right">
+              <ds-chip size="base" :color="formData.groupType === '' ? 'danger' : 'medium'">
+                <base-icon v-if="formData.groupType === ''" name="warning" />
+              </ds-chip>
+            </ds-text>
```

Comment:
I would not show this chip if there is no error.
Or I would show a grey `1 / 1` if there is no error and not an empty chip.

Same on action radius …

Reply:
done